.golangci.yml

```diff
@@ -57,81 +80,51 @@ linters-settings:
   gosec:
     excludes:
       - G115
-      - G301 # Fixable
-      - G302 # Fixable
-      - G306 # Fixable
+      - G204
```

Comment:
Why are we excluding `G204`? 

Reply:
G204 was included in exclude-rules list before, but on the current linter version we are getting too many new gosec errors, so decided to put them into one place- general config, not individually

Reply:
OK, add a todo and refer to the issue where we plan to fix these issues.

Reply:
Done